extras/Makefile.am

```diff
@@ -7,11 +7,15 @@ CLEANFILES = *~
 install-data-local:
 	$(MKDIR_P) $(DESTDIR)$(GLUSTER_BLOCKD_LIBEXECDIR);                           \
 	$(INSTALL_DATA) -m 755 $(top_srcdir)/extras/wait-for-bricks.sh               \
-		$(DESTDIR)$(GLUSTER_BLOCKD_LIBEXECDIR)/wait-for-bricks.sh;                 \
+		$(DESTDIR)$(GLUSTER_BLOCKD_LIBEXECDIR)/wait-for-bricks.sh;               \
```

Comment:
probably this line has tabs some where and which is why it is not aligning.

Reply:
This should be okay, since the line 9 and line 10 is only one command.
And for this line I just add some space before '\' to align it to the others.